### ISSUE TITLE
[Suggestion] Display compatible Forge versions on download sites

### ISSUE THREAD

**Issue by realjackasterisk** [sp614x/optifine#3247]
Concept using some good old Inspect Element:

![image](https://user-images.githubusercontent.com/31686191/72870489-354d1e80-3c9d-11ea-8677-73bd650db0a3.png)

This would just help save a LOT of hassle for new users. It'd be a nice time saver for regular users, too.

The only problem I see are the semi-recent "internally compatible" versions. For example, the changelog for 1.14.4 F3 states the following:
```
 - internally compatible with Forge 28.0.23 (can't be installed yet)
```

I'm not yet sure what kind of short message you could use for cases like this.

**Comment by asbyth** [comment]
internally compatible could just be replaced with not compatible, as it doesn't really matter to the user if its internally compatible or not

**Comment by realjackasterisk** [comment]
That's true, and that was pretty much my thinking when I initially made the concept, but on second thought I'm not sure if it should be outright misleading. It's *technically* compatible, but it cannot be installed and used via Forge. Personally, I think there should be a separate message that conveys that somehow. 

Surely there's a reason @sp614x put that in the changelog in the first place, right?

**Comment by sp614x** [comment]
This is in the chngelog to show that all necessary Forge patches are inside, but the mod launcher that Forge uses can't load OptiFine corectly.
